codes/swift/chapter_computational_complexity/space_complexity/space_complexity.swift

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Could we put the file in the first level of the folder, i.e., `./swift/chapter_computational_complexity/space_complexity.swift`? It will help keep the consistency with other languages.